etcdserver/raft.go

```diff
@@ -226,6 +226,10 @@ func (r *raftNode) start(s *EtcdServer) {
 				if err := r.storage.Save(rd.HardState, rd.Entries); err != nil {
 					plog.Fatalf("raft save state and entries error: %v", err)
 				}
+				if !raft.IsEmptyHardState(rd.HardState) {
+					proposalsCommitted.Set(float64(rd.HardState.Commit))
```

Comment:
That sounds very much like a gauge then, as it doesn't start at 0 per process.

It's the same way we'd consider the size of a append-only file to be a gauge.